roles/redpanda_broker/defaults/main.yml

```diff
@@ -26,3 +29,25 @@ node_exporter_version: "{{ node_exporter_custom_version | default('1.5.0') }}"
 cloud_storage_credentials_source: "config_file"
 cloud_storage_enable_remote_write: "true"
 cloud_storage_enable_remote_read: "true"
+rp_key_rpm: "https://dl.redpanda.com/public/redpanda/gpg.988A7B0A4918BC85.key"
+rp_config_url_rpm: "https://dl.redpanda.com/public/redpanda/config.rpm.txt"
+rp_conf_loc_rpm: "/etc/yum.repos.d/redpanda-redpanda.repo"
+rp_key_deb: "https://dl.redpanda.com/sMIXnoa7DK12JW4A/redpanda/gpg.988A7B0A4918BC85.key"
+rp_config_url_deb: "https://dl.redpanda.com/sMIXnoa7DK12JW4A/redpanda/config.deb.txt"
```

Comment:
given we're doing this in part for better airgap support, we should break these urls into a host and path component so the user only has to modify one variable.

also: the gpg keys are the same I think, so we could probably reduce this down to one rp_key variable. 

not sure why the paths would be different anyway (public vs sMIXnoa7DK12JW4A)

Reply:
I'm not sure either so I went with Chesterton's fence and kept both keys LOL

I've just pushed a commit that adds a base url element